### ISSUE TITLE
Warn about <field-def> with no name attriute

### ISSUE THREAD

**Issue by da2x** [leonardschardijn/Chirurgien#7]
Should raise import error.

**Comment by leonardschardijn** [comment]
Yeah, I was thinking of adding a sort of XML definition semantic validation check at some point. Issuing warnings that undefined IDs are being used, that there are unnamed fields and that sort of thing.

Currently, the XML definition validation is only syntactic.
